.github/workflows/bridge-ui.yml

```diff
@@ -0,0 +1,36 @@
+name: Bridge UI CI/CD
+
+on:
+  push:
+    branches-ignore:
+      - release-please-*
+    paths:
+        - "packages/bridge-ui/**"
+
+jobs:
+  deploy-bridge-ui-preview: 
+    if: ${{ !github.ref_name == 'main' }}  
+    uses: ./.github/workflows/bridge-ui-deploy.yml  
+    with:
+      environment: 'preview'
+      flags: ''
+      vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+      vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID_BRIDGE_UI }}
+
+  deploy-bridge-ui-preview-internal: 
+    if: ${{ !github.ref_name == 'main' }}
+    uses: ./.github/workflows/bridge-ui-deploy.yml
+    with:
+      environment: 'preview'
+      flags: ''
+      vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+      vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID_BRIDGE_UI_INTERNAL }}
+
+  deploy-bridge-ui-production: 
+    if: ${{ github.ref_name == 'main' && startsWith(github.ref, 'refs/tags/bridge-ui-') }}
+    uses: ./.github/workflows/bridge-ui-deploy.yml
+    with:
+      environment: 'production'
+      flags: '--prod'
+      vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+      vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID_BRIDGE_UI }}
```

Comment:
can you install editorconfig? your editor should add a new line at EOF. otherwise lgtm

Reply:
ref: https://github.com/taikoxyz/taiko-mono/blob/main/.editorconfig#L8

Reply:
Installed.